presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorAccessControl.java

```diff
@@ -53,280 +52,280 @@
 public interface ConnectorAccessControl
 {
     /**
-     * Check if identity is allowed to create the specified schema in this catalog.
+     * Check if session is allowed to create the specified schema in this catalog.
```

Comment:
Saying *session* here feels wrong, maybe it because it implies we are making a decision on something based other than identity. I'd rather keep the original word *identity*.